ocp_resources/cdi_config.py

```diff
@@ -56,6 +56,11 @@ def wait_until_upload_url_changed(self, uploadproxy_url, timeout=TIMEOUT_4MINUTE
                     return
 
     def is_garbage_collector_enabled(self):
-        """Check if garbage collector enabled on CDIconfig resource"""
+        """
+        Check if garbage collector enabled on CDIconfig resource
+
+        Returns:
+            True if the dataVolumeTTLSeconds don't exists or if dataVolumeTTLSeconds > 0
```

Comment:
`bool: True if the dataVolumeTTLSeconds don't exists or if dataVolumeTTLSeconds > 0`